### PR TITLE
Initial addition of svgwrite recipe.

### PR DESCRIPTION
**Pull request by jeromekelleher**
This is my first PR, so apologies for any mistakes. Svgwrite is a pure-python SVG library.

### CODE REVIEW

**Comment by jeromekelleher** [comment]
@conda-forge-admin, please lint

**Comment by jeromekelleher** [comment]
@conda-forge-admin, please lint

**Comment by jeromekelleher** [comment]
Notes for myself:

- I can't figure out why linting is failing. I've tried comparing with other packages that are passing, but can't see anything obvious.
- It's also mysterious why tests are failing across the CI providers. All the files required seem to be present, but when the package is tested the 'svgwrite' package is missing. When I build the package locally, it installs fine and I can import svgwrite.

File: recipes/svgwrite/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  host:
```
Comment (isuruf) [inline recipes/svgwrite/meta.yaml]: Change this to `build`.

Comment (jeromekelleher) [reply recipes/svgwrite/meta.yaml]: Thanks @isuruf, greatly appreciated!

**Comment by jeromekelleher** [comment]
Hi @conda-forge/staged-recipes. I think this is ready to merge? Thanks!

**Comment by jeromekelleher** [comment]
Not sure if I got the @ notation right here, retrying:

Hi @conda-forge/staged-recipes 
I think this is ready to merge? Thanks!

**Comment by bgruening** [comment]
Thanks @jeromekelleher!

**Pull request merged**